presentation/src/main/java/com/yapp/bol/presentation/view/match/game_result/GameResultAdapter.kt

```diff
@@ -0,0 +1,104 @@
+package com.yapp.bol.presentation.view.match.game_result
+
+import android.content.Context
+import android.graphics.Color
+import android.view.KeyEvent
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.EditText
+import androidx.core.widget.doAfterTextChanged
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.yapp.bol.presentation.R
+import com.yapp.bol.presentation.databinding.RvPlayerMatchItemBinding
+import com.yapp.bol.presentation.model.MemberResultItem
+import com.yapp.bol.presentation.utils.Constant.EMPTY_STRING
+
+class GameResultAdapter(
+    private val context: Context,
+    private val gameResultUpdateListener: GameResultUpdateListener,
+) : ListAdapter<MemberResultItem, GameResultAdapter.GameResultViewHolder>(diff) {
+
+    interface GameResultUpdateListener {
+        fun updatePlayerScore(position: Int, value: Int?)
+        fun sortPlayerScore()
+        fun showKeyboard(editText: EditText)
+        fun moveScroll(position: Int)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): GameResultViewHolder {
+        val binding =
+            RvPlayerMatchItemBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return GameResultViewHolder(binding, context, gameResultUpdateListener)
+    }
+
+    override fun onBindViewHolder(holder: GameResultViewHolder, position: Int) {
+        holder.bind(getItem(position), position)
+    }
+
+    override fun getItemViewType(position: Int) = position
+
+    override fun getItemId(position: Int) = position.toLong()
+    class GameResultViewHolder(
+        private val binding: RvPlayerMatchItemBinding,
+        private val context: Context,
+        private val gameResultUpdateListener: GameResultUpdateListener,
+    ) : RecyclerView.ViewHolder(binding.root) {
+
+        fun bind(item: MemberResultItem, position: Int) {
+
+            setTextView(item)
+            setClickListener(position)
+            setTextChangeListener(position)
+        }
+
+        private fun setTextView(item: MemberResultItem) {
+            binding.tvMemberRank.text =
+                String.format(context.resources.getString(R.string.game_result_rank), item.rank + 1)
+            if (item.rank == 0) binding.tvMemberRank.setTextColor(Color.parseColor("#FF4D0D"))
+            binding.tvMemberName.text = item.name
+            val score = if (item.score == null) EMPTY_STRING else item.score.toString()
```

Comment:
```suggestion
            val score = item.score.toString()
```
요렇게 해도 의도하신 동작일 것 같아요.